### ISSUE TITLE
In solo, Virus did nothing

### ISSUE THREAD

**Issue by kevinb9n** [bafolts/terraforming-mars#2302]
In a solo game, I played Virus, and it didn't ask me whether I wanted to remove the nonexistent opponent's nonexistent animals or the nonexistent opponent's nonexistent plants. It just did nothing. As a result, my Crash Site Cleanup card remained as unplayable as before.

I *think* this should work. I see no official ruling, but this person's reasoning seems sound to me:
https://boardgamegeek.com/thread/2452083/article/35163701#35163701

**Comment by vsrisuknimit** [comment]
@kevinb9n can you try to get an official ruling?

**Comment by kevinb9n** [comment]
Okay, I necro'd the thread.

If we don't hear anything for a while, I do think the behavior I described is what users who know the rest of the rules well will be expecting, so a change could be justified for that reason. Note that if I'd been Mons Insurance I would have had to pay 3 MC when I played Virus (does anyone know if the app is getting that part right?) so it's self-inconsistent for it to not power up CSC too.
